packages/backend/src/connector/core/middleware/error-handler.ts

```diff
@@ -26,7 +26,10 @@ export function createIncomingErrorHandlerMiddleware(
       if (ctx.revertTotalReceived) {
         await ctx.revertTotalReceived()
       }
-      ctx.response.reject = errorToIlpReject(serverAddress, err)
+      ctx.response.reject = errorToIlpReject(
+        serverAddress,
+        err as IlpErrorClass
```

Comment:
I'm wondering if this will still be necessary with the latest `ilp-packet`
- https://github.com/interledger/rafiki/pull/775

Should we merge that and see what happens here?

At the very least, we could use the new `isIlpError` type guard
https://github.com/interledgerjs/interledgerjs/blob/ba41fb3e081b4fa0a6c8a26886ca415663f195f2/packages/ilp-packet/src/index.ts#L173

Reply:
Sure, let's do that. I was holding off on merging the `alpha` package updates but if it helps, we should just merge them.

Reply:
Please check that I updated that correctly.